CGPs/CGP-0044.md

```diff
@@ -0,0 +1,48 @@
+---
+cgp: 44
+title: Increasing Mento bucket sizes
+date-created: 2021-10-12
+author: @martinvol, @rcroessmann
+status: DRAFT
+discussions-to: 
+governance-proposal-id: [if submitted]
+date-executed: [if executed] <date created on, in ISO 8601 (yyyy-mm-dd) format>
+
+---
+## Overview
+
+This Governance intends to increase the amount of the buckets, as a percent of the total amount of the unfrozen Reserve Celo buckets. The concrete proposal is to lift the current bucket sizes to `X%`.
+
+Updating the bucket sizes means that Mento has more liquidity to provide to arbitragers, as well as making exchanges suffer less slippage, which could lead to a tighter peg. 
+
+### Status
+  
+Currently, the parameter `reserveFraction` is set to 0.5%. This paramter was sufficent in the past, but as demand for Celo Stable assets growing in the last months, this parameter could be updated to be able to provide greater liquidity.
```

Comment:
typo `paramter` -> `parameter`
grammar: `assets growing` -> `assets is growing`
`last months` -> `last few months`